src/main/resources/org/datadog/jenkins/plugins/datadog/DataDogJobProperty/help-tagging.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    <p>A newline separated list of tags to push to DataDog. Tags MUST follow the format <em>key:value</em>.<p>
+    <p>Example: <em>os=${OS_PARAM}</em>.</p> 
+    <p>The value can either be something that is automatically expanded in Jenkins, or a static value which you define for the tag</p>
+    <p>The OS_PARAM in this case is a value that is expanded by Jenkins (Environment and/or build variable)</p>
```

Comment:
These two lines could be condensed to say, 

`<p>The value can either be something that is automatically expanded by Jenkins (environment and/or build variable), or a static value. The OS_PARAM in this case is expanded by Jenkins.</p>`